data/json/mutations/mutations.json

```diff
@@ -8948,6 +8954,7 @@
     "vitamin_cost": 60,
     "description": "Your flesh is highly poisonous, and creatures that bite you will receive a nasty surprise.",
     "category": [ "FISH", "GASTROPOD" ],
+    "flags": [ "IRRITANT_IMMUNE" ],
```

Comment:
```suggestion
```

Reply:
I'm not sure why I put it for that one, it wouldn't do anything against chemicals.